fairing/deployers/tfjob/tfjob.py

```diff
@@ -18,7 +17,10 @@ def __init__(self, namespace=None, worker_count=1, ps_count=0,
 
     def create_resource(self):
         self.created_tfjob = self.backend.create_tf_job(self.namespace, self.deployment_spec)
-        return self.created_tfjob['metadata']['name']
+        if self.created_tfjob:
+            return self.created_tfjob['metadata']['name']
+        else:
+            raise RuntimeError("Error happens while creating TFJob.")
```

Comment:
No need to handle the case here. See my comment below.

Reply:
Updated, thanks